bin/test/test_prerequisites_minus_squid.sh

```diff
@@ -0,0 +1,97 @@
+            echo "$(date) - ${count} try - ${label} not listening${color_reset} on ${port} yet"
+            exit_message="${color_red}$(date) - ${label} not listening${color_reset} on ${port}"
+            exit_code=1
+        fi
+        sleep 1
+    done
+    echo -e "\n${exit_message}"
+    return "${exit_code}"
+}
+
+function check_unix_socket(){
+    # vars
+    label="${1:-}"
+    socket_file="${2:-}"
+    path="${3:-}"
+
+    # usage
+    if [ -z "${label}" ] || [ -z "${socket_file}" ] ; then
+        echo "${color_red}Usage:${color_reset} check_unix_socket {label} {socket_file} {command}"
+        return 1
+    fi
+
+    for count in $(seq 30) ; do
+        if echo -e "${path}" | nc -U "${socket_file}" ; then
+            exit_message="${color_green}$(date) - ${label} listening${color_reset} on ${socket_file}"
+            exit_code=0
+            break
+        else
+            echo "$(date) - ${count} try - ${label} not listening${color_reset} on ${socket_file} yet"
+            exit_message="${color_red}$(date) - ${label} not listening${color_reset} on ${socket_file}"
+            exit_code=1
+        fi
+        sleep 1
+    done
+    echo -e "\n${exit_message}"
+    return "${exit_code}"
+}
+
+function cat_and_exit_err() {
+    file="$1"
+    cat "$file"
+    return 1
+}
+
+echo -e "\n----- install servers prerequisites -----"
+python3 -m pip install --requirement bin/requirements-frozen.txt
+
+echo -e "\n----- start servers -----"
+cd integration/hurl
+mkdir -p build
+
+echo -e "\n------------------ Starting server.py"
+python3 server.py > build/server.log 2>&1 &
+check_listen_port "server.py" 8000 || cat_and_exit_err build/server.log
+
+echo -e "\n------------------ Starting tests_ssl/ssl_server.py (Self-signed certificate)"
+python3 tests_ssl/ssl_server.py 8001 tests_ssl/certs/server/cert.selfsigned.pem false > build/server-ssl-selfsigned.log 2>&1 &
+check_listen_port "tests_ssl/ssl_server.py" 8001 || cat_and_exit_err build/server-ssl-selfsigned.log
+
+echo -e "\n------------------ Starting tests_ssl/ssl_server.py (Signed by CA)"
+python3 tests_ssl/ssl_server.py 8002 tests_ssl/certs/server/cert.pem false > build/server-ssl-signedbyca.log 2>&1 &
+check_listen_port "tests_ssl/ssl_server.py" 8002 || cat_and_exit_err build/server-ssl-signedbyca.log
+
+echo -e "\n------------------ Starting ssl/ssl_server.py (Self-signed certificate + Client certificate authentication)"
+python3 tests_ssl/ssl_server.py 8003 tests_ssl/certs/server/cert.selfsigned.pem true > build/server-ssl-client-authent.log 2>&1 &
+check_listen_port "tests_ssl/ssl_server.py" 8003 || cat_and_exit_err build/server-ssl-client-authent.log
+
+echo -e "\n------------------ Starting tests_unix_socket/unix_socket_server.py"
+python3 tests_unix_socket/unix_socket_server.py > build/server-unix-socket.log 2>&1 &
+check_unix_socket "tests_unix_socket/unix_socket_server.py" build/unix_socket.sock "GET /hello HTTP/1.0\r\n"
+
+# Squid must be running via Docker Compose
```

Comment:
I created this version because the other one uses `sudo` in a few places, and I wanted to delegate the `squid` piece to a compose stack instead.